binaryornot/check.py

```diff
@@ -8,8 +8,9 @@
 """
 
 import logging
+import argparse
```

Comment:
What about sorting these imports?

Reply:
Agreed. That's a nice touch for projects.